euphoria-examples/src/main/java/cz/seznam/euphoria/examples/wordcount/AccessLogCount.java

```diff
@@ -0,0 +1,140 @@
+package cz.seznam.euphoria.examples.wordcount;
```

Comment:
may i ask for the usual the usual license header? otherwise it looks quite good to me. many thanks for sharing! :+1: